setup.cfg

```diff
@@ -9,6 +9,5 @@ doctests = 1
 [check-manifest]
 ignore =
     .editorconfig
-    .meta.cfg
+    .meta.toml
     docs/_build/html/_sources/*
-    docs/_build/doctest/*
```

Comment:
Hm.  It's probably fine, right?

Reply:
There are not doctests run using sphinx so this path does not exist.